session4/app.js

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import ReactDom from "react-dom/client";
+import { Data } from "./data.js";
+import Card from "./cardcomponent.js";
+
+root = ReactDom.createRoot(document.getElementById("root"));
+
+const HeadingWithSeachBar = () => {
+  return (
+    <div className="header">
```

Comment:
Can you keep Heading and Search bar in different component?
I might not want to use SearchBar everywhere but might want to use Heading everywhere.